src/lifedrain/lifedrain.py

```diff
@@ -723,14 +750,17 @@ def setAnkiProgressBarStyle(self, config=None):
         if self._currentDeck is not None:
             self.recover(value=0)
 
-    def recover(self, increment=True, value=None):
+    def recover(self, increment=True, value=None, damage=False):
         '''
         Abstraction for recovering life, increments the bar if increment is True (default).
         '''
         multiplier = 1
         if not increment:
             multiplier = -1
-        if value is None:
+        if value is None and damage and self._barInfo[self._currentDeck]['enableDamageValue']:
```

Comment:
Sorry for being picky, but here we're checking for `value is None` twice, can't you change the logic a little so it checks only once? Something like this would be nice:

```python
if value is None:
    if damage and self._barInfo[self._currentDeck]['enableDamageValue']:
        # Your implementation
    else:
        # The default behavior (set multiplier and value)
        # As multiplier is not used in your implementation, we can move the code that sets it here.
        multiplier = 1 if increment else -1  # <-- We can make my implementation shorter using this
        # ...
```

Reply:
You're entirely right, that would be better (even though it introduces an ugly scope pyramid :p) Thanks!